mlflow/server/js/src/experiment-tracking/components/CompareRunBox.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
```

Comment:
Can you add unit tests for this component?

Reply:
Ok sure.